### ISSUE TITLE
Crashes on iOS Safari

### ISSUE THREAD

**Issue by tom-leamon** [gabrielpetersson/fast-grid#4]
The demo crashes within a couple seconds on iPhone due to memory usage issues.

**Comment by gabrielpetersson** [comment]
which iphone? it loads 100k rows by default which should be fine for any phone even older ones

**Comment by tom-leamon** [comment]
Technically it's not an iPhone issue, its a Safari limitation. On iOS Safari, memory for a single tab is limited to 384MB. Yes, it's ridiculous but it's also reality. Consider doing a user agent check and making sure not to load too much memory so it can work on iPhone.

**Comment by gabrielpetersson** [comment]
makes sense - iphone shouldn't use the off-thread stuff cause it's a bit memory intensive due to js inability to work on objects between threads
